.travis.yml

```diff
@@ -26,14 +36,16 @@ before_deploy:
   - mkdir deploy
   - if [ $TRAVIS_OS_NAME = osx ]; then cp libopenpnp-capture.dylib deploy; cp mac/tests/openpnp-capture-test deploy/openpnp-capture-test-mac; fi
   - if [ $TRAVIS_OS_NAME = linux ]; then cp libopenpnp-capture.so deploy; cp linux/tests/openpnp-capture-test deploy/openpnp-capture-test-linux; fi
-
+  - if [ $TRAVIS_CPU_ARCH = arm64 ]; then cp libopenpnp-capture.so deploy/libopenpnp-capture-arm64.so; cp linux/tests/openpnp-capture-test deploy/openpnp-capture-test-linux-arm64; fi
 deploy:
   provider: releases
-  api_key:
+  token:
     secure: Vep5q/3rmKlU1bwi4R7/vrOOvOzZ/S+ukNBgjioMWMQP+f/QFZPc1IO0VP3kqxuxpvG4rWJR0djSu+FJG6ptK9OZbjVZvEAiBJVQZnHkDHyc8Gye2GCj/4XPUWiKXVQS5fTFHI7NenGPMZlRs/N9xI156gwiOLkpPK2Di1Zna51Mig74zU68WpDRsVWNlKu2nIx1WK2W7fWn9IM1FrtM86BPnztkqHUOBKhyYTpw7hKtAnqiB/09u5vBqd0DeoTohYaSAtFcKeNVBFtZO5wF+VKRnDDRMitM3SgEFElHT68KRR9zoqEvdmieK2GsoEhNzOKWRhjBWG1JvxJOlish8m1gftOiRPmNKq+PrLLReZpIRh7XoqXhmSW2M5BzrWnv1l957tRp0V/2dy6gZudCD4Kwh3CV/XXfUK6J6yPaCpCqSGV2ylHoX9hhVMheh4kdjpKUi4dsfQ5suLqgSIwge6trBjwyXUqrsnH9KAZJEpLCfIR4E870phOV5nuNkeFqS5mFdqJIlJPjORFzNgbJ5fx9yH9RqIHEuTH5Rkd8svPWKo7KU9L1wJcjm2k1VHqy5MCxenMGuRafUNAOQ9PoiAd7ijr/zvrUgvEjl353rWheTYllkRovt4C6DQ08fcC5w2BlofuRWWk/+Z69Pd0QkB7+dqae92csshaxe1TvAHA=
   file_glob: true
-  file: deploy/*    
-  skip_cleanup: true
+  file: deploy/*
+  cleanup: false
+  edge: true
```

Comment:
Okay, it looks like this selects the beta version of the provider, which explains the above questions. Is this required for your deployment to work? If it's not, I don't see a good reason to switch to a beta provider at this time.

Reply:
Yes, all the changes above are informed by TravisCI support, `travis lint`'s CLI and web linter too:

https://config.travis-ci.com/explore

See how your current `.travis.yml` on your `master` looks like against the official linter:

![Skärmavbild_2020-05-07_kl__09_17_17](https://user-images.githubusercontent.com/175587/81237777-c628e980-9043-11ea-99ad-63b327cef06b.png)

I really just wanted to future-proof this a little bit, but happy to revert back to the old syntax.